compiler/parser/RdrHsSyn.hs

```diff
@@ -569,6 +582,52 @@ done.
 
 -}
 
+splitCon :: [LHsType GhcPs]
```

Comment:
For my understanding: Was this function written by us? Or was it introduced in a later commit to GHC and we are backporting it because we rely on it?

Reply:
This function is in fact from "way back when" in GHC. We forward ported it from an older version to newer versions. Ideally we would work out what it should now be replaced with and remove this function accordingly.

Reply:
Ah, I see. Thanks